### DISCUSSION TITLE
cannot extract text from a pdf1.3(acrobat 4.x) file

### DISCUSSION THREAD

**Discussion by liutianzhu** [jsvine/pdfplumber#337] [Ask for help with specific PDFs]
[1.pdf](https://github.com/jsvine/pdfplumber/files/5834899/1.pdf)

simple code as:

`import pdfplumber`
`path = "1.pdf"`
`p = pdfplumber.open(path)`
`p1 = p.pages[0]`
`t = p1.extract_text()`
`print(t)`

run it, and result as:

`BOOKING REQUEST`
`°`

It just get the first few letters, I guess it might be sth with the font?

**Comment by jsvine** [comment]
Yes, it does seem to be caused by the font: `STSong-Light`, which this PDF primarily uses, does not appear to be extractable under the current version of `pdfminer.six`, which this library depends on. See this issue for details, and to follow along: https://github.com/jsvine/pdfplumber/issues/332

**Reply by liutianzhu** [reply]
Got it. Thank you so much.
